components/issue-proposals.tsx

```diff
@@ -32,53 +34,167 @@ export default function IssueProposals({ numberProposals, issueId, amount }) {
-              </p>
-              <button
-                className="btn btn-md btn-purple"
-                onClick={() => handleDispute(toNumber(proposal._id))}
-              >
-                Dispute
-              </button>
+              <div className="d-flex flex-column bd-highlight mr-2">
+                <div className="d-flex align-items-stretch mb-0 ">
+                  <p
+                    className={clsx("smallCaption mb-0", {
+                      "color-red": proposal?.isDisputed,
+                      "color-purple": !proposal?.isDisputed,
+                    })}
+                  >
+                    {proposal.disputes}
+                  </p>
+                  <p className="smallCaption mb-0">/{amount} Oracles</p>
+                </div>
+                <div className="content-relative">
+                  <div className="progress progress-oracle my-1">
+                    <div
+                      className={clsx("progress-bar ", {
+                        "bg-red": proposal?.isDisputed,
+                        "bg-purple": !proposal?.isDisputed,
+                      })}
+                      role="progressbar"
+                      style={{
+                        width: `${handlePercentage(
+                          toNumber(proposal.disputes),
+                          amount
+                        )}%`,
+                      }}
+                    >
+                      <div className="progress progress-ball left">
+                        <div
+                          className={clsx("progress-bar", {
+                            "bg-red": proposal?.isDisputed,
+                            "bg-purple": !proposal?.isDisputed,
+                          })}
+                          role="progressbar"
+                          style={{
+                            width: `${handlePercentage(
+                              toNumber(proposal.disputes),
+                              (amount * 7) / 100
+                            )}%`,
+                          }}
+                        ></div>
+                      </div>
+                      <div className="progress progress-ball right">
+                        <div
+                          className={clsx("progress-bar", {
+                            "bg-red": proposal?.isDisputed,
+                            "bg-purple": !proposal?.isDisputed,
+                          })}
+                          role="progressbar"
+                          style={{
+                            width: `${
+                              handlePercentage(
+                                toNumber(proposal.disputes),
+                                amount
+                              ) > 81.6 &&
+                              ((handlePercentage(
+                                toNumber(proposal.disputes),
+                                amount
+                              ) -
+                                81.7) *
```

Comment:
please, if these numbers are constants, set them as so at the top of the file.

Reply:
ok, i adjusting